### DISCUSSION TITLE
check DRC on a routed design

### DISCUSSION THREAD

**Discussion by seshu-lyptus** [The-OpenROAD-Project/OpenROAD-flow-scripts#1847] [Q&A]
Hi
I have a routed design ODB and would like to check the  DRC 
Is there any standalone command for checking the DRC 
how can i use the developer function like check_drc

**Comment by maliberty** [comment]
OR is not a general purpose drc tool - magic or klayout are better for that.  The OR detailed router does provide some LEF based checking, but only for signal nets.

**Comment by seshu-lyptus** [comment]
Thanks 
Is there an inbuilt command or procedure to  perform the LEF based rule checking ?

**Comment by maliberty** [comment]
check_drc as you already mentioned

**Comment by seshu-lyptus** [comment]
Following is the snapshot , might be my OpenROAD is not latest , or am i missing any steps
![Screenshot 2024-03-17 082232](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/159769472/5fc0119f-e2ac-4be5-81d6-1340bdecaeaf)

**Reply by maliberty** [reply]
As it is not a user command it isn't in the general namespace, use `drt::check_drc`

**Comment by seshu-lyptus** [comment]
Thanks for a quick response 
It worked
